### ISSUE TITLE
Fix San Diego

### ISSUE THREAD

**Issue by justinelliotmeyers** [openaddresses/openaddresses#4875]
seems like we had a similar source a long time ago, then they must have switched. Then we had a one and done download I think that always pulled the same amount perhaps? Eitherwise last time it broke. Found a new source.
Broken:
http://results.openaddresses.io/sources/us/ca/san_diego
New:
https://gissd.sandag.org/rdw/rest/services/Address/Address/MapServer/0

**Comment by orangejulius** [comment]
Looks like the new source is already being used as of https://github.com/openaddresses/openaddresses/pull/4244

https://github.com/openaddresses/openaddresses/blob/2f6a969212bc9b854fc53231cf5687edd386c8f7/sources/us/ca/san_diego.json#L13

Even that pull request mentions slowness from the source though, so maybe the servers are particularly flaky for San Diego?

**Comment by justinelliotmeyers** [comment]
okay, should I close this out?  Thanks for looking at this for me!!

**Comment by orangejulius** [comment]
Taking a look at a recent log for the San Diego source, I see errors like this:

```
2020-05-26 03:15:47,706  WARNING: Could not download ESRI source data: ('Could not connect to URL', EsriDownloadError('Could not retrieve this chunk of objects: Error performing query operation ',))
2020-05-26 03:15:47,706  WARNING: Error in process_one.process()
Traceback (most recent call last):
  File "/usr/local/lib/python3.5/dist-packages/esridump/dumper.py", line 366, in __iter__
    data = self._handle_esri_errors(response, "Could not retrieve this chunk of objects")
  File "/usr/local/lib/python3.5/dist-packages/esridump/dumper.py", line 76, in _handle_esri_errors
    ', '.join(error['details']),
esridump.errors.EsriDownloadError: Could not retrieve this chunk of objects: Error performing query operation 

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.5/dist-packages/openaddr/process_one.py", line 134, in process
    cache_result = cache(layer + '-' + data_source['name'], data_source, temp_dir, extras)
  File "/usr/local/lib/python3.5/dist-packages/openaddr/__init__.py", line 97, in cache
    downloaded_files = task.download(source_urls, workdir, data_source.get('conform'))
  File "/usr/local/lib/python3.5/dist-packages/openaddr/cache.py", line 392, in download
    for feature in downloader:
  File "/usr/local/lib/python3.5/dist-packages/esridump/dumper.py", line 372, in __iter__
    raise EsriDownloadError("Could not connect to URL", e)
esridump.errors.EsriDownloadError: ('Could not connect to URL', EsriDownloadError('Could not retrieve this chunk of objects: Error performing query operation ',))
```

If there's something we can do about that, it would be nice to get the latest data from San Diego (last successful run was in February 2020).

But if we think it's just a matter of waiting until the run eventually passes, then I'd say we're good here.

Let me know if I can help with any of it.

**Comment by iandees** [comment]
Poking at the /query endpoint of the service makes it look like they have a *very* slow or misconfigured backend database. It's unlikely we'll be able to make it work.

**Comment by justinelliotmeyers** [comment]
I'll have to look at this more. Not sure if https://data.sandiego.gov/datasets/address-points-apn/ helps. Will dl tonight and compare to what we have. Otherwise, I can call and try to get this sorted out.

**Comment by iandees** [comment]
The link you pointed to is used in the City of San Diego source here: https://github.com/openaddresses/openaddresses/blob/master/sources/us/ca/city_of_san_diego.json

I wonder if the "san_diego.json" source really covered the whole county or just the city.

**Comment by arch0345** [comment]
Resolved in [#6280](https://github.com/openaddresses/openaddresses/pull/6280)
